packages/spatial/src/renderer/components/DirectionalLightComponent.ts

```diff
@@ -36,13 +38,19 @@ import {
 import { useEntityContext } from '@etherealengine/ecs/src/EntityFunctions'
 import { matches, useMutableState } from '@etherealengine/hyperflux'
 
+import { EntityUUID, UUIDComponent, defineQuery } from '@etherealengine/ecs'
+import { RenderSettingsComponent } from '@etherealengine/engine/src/scene/components/RenderSettingsComponent'
```

Comment:
spatial package cannot access RenderSettingsComponent. cleaning up the CSM should not be the responsibility of the DirectionalLightComponent, we should implement this in the shadow system